callautomation-live-transcription/websocket.py

```diff
@@ -0,0 +1,23 @@
+import asyncio
+import json
+import websockets
+
+async def handle_client(websocket, path):
+    print("Client connected")
+    try:
```

Comment:
Are we logging TranscriptionData and TranscriptionMetaData, similar to how we do it for C# quick start ?

Reply:
Updated logs for transcription metadata and data.